setup.py

```diff
@@ -7,13 +7,13 @@
 from setuptools.command.test import test as TestCommand
 
 tests_require = (
-    'pytest>=6.2.5,<7.2',
+    'pytest>=6.2.5,<7.5',
     'pytest-django',
 )
 
 
 install_requires = (
-    'Django>=2.2,<3.3',
+    'Django>=3.2,<=4.2',
```

Comment:
Would we want this to be <=4.3?

Reply:
Later I realized I should have used <4.3, is that okay with you? I've seen that we mostly use this as a convention